.github/workflows/ci.yml

```diff
@@ -35,12 +35,29 @@ jobs:
       - name: Run pipeline with test data
         run: |
           nextflow run ${GITHUB_WORKSPACE} -profile test,docker
-      - name: Run pipeline with multi test data
-        run: | 
-          nextflow run ${GITHUB_WORKSPACE} -profile test_multi,docker
-      - name: Run pipeline with ITS PacBio reads and Unite database
-        run: | 
-          nextflow run ${GITHUB_WORKSPACE} -profile test_pacbio_its,docker
-      - name: Run pipeline with double cutadapt steps on test data with double primers
+
+  profiles:
+    name: Run workflow profile
+    # Only run on push if this is the nf-core dev branch (merged PRs)
+    if: ${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/mag') }}
```

Comment:
Is it supposed to be nf-core/mag?

Reply:
uuups good catch, I copied it from mag, I'll fix it

Reply:
Fixed